src/Stratis.Bitcoin/Base/BaseFeature.cs

```diff
@@ -233,6 +239,9 @@ public static void BuildDefaultConfigurationFile(StringBuilder builder, Network
                 Directory.CreateDirectory(this.dataFolder.ChainPath);
             }
 
+            this.logger.LogInformation("Loading finalized block height");
+            await this.finalizedBlockHeight.LoadFinalizedBlockHeightAsync().ConfigureAwait(false);
```

Comment:
Yes thats good

Reply:
I don't think we should but information on the console about this when we are running bitcoin which does not have any finalised block height.

Reply:
Technically it does- the genesis block 😄

Reply:
Today I learned something new